src/ToastContainer.js

```diff
@@ -282,18 +283,14 @@ class ToastContainer extends Component {
           {...this.getContainerProps(position, disablePointer)}
           key={`container-${position}`}
         >
-          {toastToRender[position].map(item => item)}
+          {toastToRender[position]}
         </TransitionGroup>
       );
     });
   }
 
   render() {
-    return (
-      <div>
-        {this.renderToast()} 
-      </div>
-    );
+    return this.renderToast();
```

Comment:
Hey @jfsiebel, just saw your comment. The backward incompatibility was caused by the fact that I removed the wrapping div.